examples/bottomhalf.c

```diff
@@ -14,6 +14,10 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 
+#ifndef DECLARE_TASKLET_OLD
```

Comment:
Add some notice from https://lwn.net/Articles/830964/

Reply:
You can put the short description in the example code. The longer one should appear in TeX document.